files/en-us/archive/web/api/canvas_api/tutorial/hit_regions_and_accessibility/index.html

```diff
@@ -0,0 +1,100 @@
+---
+title: Hit regions and accessibility
+slug: Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility
```

Comment:
This is not the archive slug, should it be?

Reply:
> This is not the archive slug, should it be?

Dunno 🤷🏻‍♂️ I’ve since read https://github.com/mdn/yari/discussions/2207, and that now makes me wonder if the archived files are supposed to have any frontmatter at all — and whether they’re supposed be a dump of the rendered HTML…